error-codes.md

```diff
@@ -356,6 +356,12 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
 
     During the execution of a non-transactional DML statement, if a batch fails, the statement is stopped. For more information, see [Non-transactional DML statements](/non-transactional-dml.md).
 
+* Error Number: 8147
+
+   When [`tidb_constraint_check_in_place_pessimistic`](/system-variables.md#tidb_constraint_check_in_place_pessimistic-new-in-v630) is set to `0`, the unique constraint checks in pessimistic transactions might not pass, causing the SQL statement execution to return this error and abort the current transaction.
+
+    When [`tidb_constraint_check_in_place_pessimistic`](/system-variables.md#tidb_constraint_check_in_place_pessimistic-new-in-v630) is set to `1`, to ensure the correctness of the transactions,  any errors in the SQL statement execution might cause TiDB to return this `8147` error and abort the current transaction. When this error occurs, you can check the error message for specific causes.
```

Comment:
The paragraph is incorrect. I've modified the Chinese version. Could you take a look at it?